packages/blade/src/components/Modal/ModalPortal.tsx

```diff
@@ -0,0 +1,24 @@
+import { createPortal } from 'react-dom';
+
+type ModalPortalProps = {
+  children: React.ReactNode;
+};
+
+const createModalRootWrapper = (id: string): HTMLElement => {
+  const element = document.createElement('div');
+  element.setAttribute('id', id);
+  document.body.appendChild(element);
+  return element;
+};
+
+const ModalPortal = ({ children }: ModalPortalProps): React.ReactElement => {
+  const wrapperID = 'modal-root-portal';
```

Comment:
Making it more specific so it doesn't conflict with any other random modal library, *just in case*
```suggestion
  const wrapperID = 'blade-modal-root-portal';
```